src/regex/pcre.cr

```diff
@@ -0,0 +1,157 @@
+require "./lib_pcre"
+
+# :nodoc:
+module Regex::PCRE
+  # :nodoc:
+  def initialize(*, _source source, _options @options)
```

Comment:
Why not make this one private too? Also, wouldn't it make sense to change its name, and avoid the renaming of arguments?

Reply:
Yeah, I think `private` should work.

Changing the name would make this not be a constructor which causes problems with ivar initialization.